libkbfs/folder_branch_status.go

```diff
@@ -240,7 +242,8 @@ func (fbsk *folderBranchStatusKeeper) getStatusWithoutJournaling(
 		}
 		_, usageBytes, archiveBytes, limitBytes,
 			gitUsageBytes, gitArchiveBytes, gitLimitBytes, quErr :=
-			fbsk.quotaUsage.GetAllTypes(ctx, 0, 0)
+			fbsk.quotaUsage.GetAllTypes(
+				ctx, quotaUsageStaleTolerance/2, quotaUsageStaleTolerance)
```

Comment:
There's a TODO above this at line 233; seems we could remove that now? Also it'd be nice to have a comment explain that for non-team TLFs we are passing in a `quotaUsage` object.

Reply:
The TODO comment still applies because it's talking about the team TLFs.  I'll make it better and include your suggestion.